### ISSUE TITLE
tests broken on Windows with latest Luarocks

### ISSUE THREAD

**Issue by fperrad** [luarocks/hererocks#9]
@hishamhm could you take a look at https://ci.appveyor.com/project/hishamhm/hererocks/builds/35418067/job/s6k82innxl02vnf4

```
AssertionError: Error running command 'test\here\bin\luarocks.bat --version': code 255, output:
b'The syntax of the command is incorrect.\r\n'
```

**Comment by hishamhm** [comment]
My hacky [bisecting-by-hand-on-appveyor](https://ci.appveyor.com/project/hishamhm/hererocks/history) seems to show that [0451337](https://github.com/luarocks/luarocks/commit/04513372d7d5859ea865252095c3b5ce0762228f) is where it starts failing. @dwenegar, any idea of what might have gone wrong?

**Comment by fperrad** [comment]
@hishamhm fixed by LuaRocks 3.5.0, thanks.
